### DISCUSSION TITLE
docker-compose thread panic

### DISCUSSION THREAD

**Discussion by undaunt** [SamTV12345/PodFetch#195] [Q&A] [answered]
I've been unable to get the container to start, whether using sqlite or postgres, without getting a thread panic and I'm really unclear to me why. I've tried removing as many variables from the equation as possible, including trying both with and without my reverse proxy, commenting out my yaml anchors, touching `touch podcast.db` and trying with no db file present, changing the SERVER_URL flag to be the local host or the reverse proxy URL, etc.

I always seem to get some variation of:

```
06/28/2023 12:45:52 PM
Debug file located at /home/rust/src/target/x86_64-unknown-linux-musl/release/build/podfetch-165e1a7f6444a2d5/out/built.rs
06/28/2023 12:46:22 PM
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Some("Unable to open the database file"))', src/main.rs:441:24
06/28/2023 12:46:22 PM
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Container stopped
```

Ubuntu 22.04 LTS
Docker version 24.0.2, build cb74dfc
Docker Compose version v2.18.1

SQLite Compose:

```
# Podfetch - Podcast Manager
  podfetch:
    image: samuel19982/podfetch
    container_name: podfetch
    #<<: *common-apps
    user: $PUID:$PGID
    volumes:
      - $DOCKERDIR:/podfetch:/app/db
      - $MEDIADIR/podcasts:/app/podcasts
    ports:
      - 8000:8000
    environment:
      - POLLING_INTERVAL=60
      - SERVER_URL=http://localhost:8000
      - DATABASE_URL=sqlite:///app/db/podcast.db
```

If I can at least get that working, I'll move on to postgres. Am I missing something very obvious here?

Thank you.

**Comment by SamTV12345** [comment, marked as answer]
The colon seems a bit off. Did you try with  - $DOCKERDIR/podfetch:/app/db . That should work.

**Comment by undaunt** [comment]
Can't believe I typoed that. Thanks for getting more eyes on it.
